vmm/src/config.rs

```diff
@@ -1154,6 +1154,11 @@ impl Drop for NetConfig {
     fn drop(&mut self) {
         if let Some(mut fds) = self.fds.take() {
             for fd in fds.drain(..) {
+                // Skip reserved FDs
```

Comment:
I don't think it should ever get to here due to other checks. But I see no harm in this.

Reply:
@rbradford I think it is required to have this check here. As long as users provide reserved FDs, a `NetConfig` instance with reserved FDs would be constructed and these reserved FDs would be closed when this instance is dropped. 

The "other checks" won't prevent this happening because such `NetCOnfig` instances are perfectly valid from Rust point of view. We can only avoid such behavior with this explicit check in the `drop` function.

Reply:
So, `NetConfig::validate` already checks that the fds are >=2. What route do you see for an fd < 2 stored in the struct ahead of a `Drop` in real use. API and CLI calls should call `.validate()` the only way I can think of for this to by bypassed is testing where `NetConfig` is manually created.

Reply:
Isn't a `NetConfig` instance already created before the call to its `.validate()`?

Reply:
So I can see why this would be required now. e.g. if we received a `NetConfig` over the wire with invalid FDs it would then fail the validation which would then lead to it going out of scope and so the `Drop` would be called which would close those invalid FDs.